src/applications/caregivers/tests/unit/components/ApplicationDownloadLink.unit.spec.jsx

```diff
@@ -38,106 +44,136 @@ describe('CG <ApplicationDownloadLink>', () => {
     localStorage.setItem('csrfToken', 'my-token');
     apiRequestStub = sinon.stub(api, 'apiRequest');
     recordEventStub = sinon.stub(recordEventModule, 'default');
-    sinon.stub(URL, 'createObjectURL').returns('my_stubbed_url.com');
-    sinon.stub(URL, 'revokeObjectURL');
   });
 
   afterEach(() => {
     localStorage.clear();
-    sinon.restore();
-  });
-
-  context('default behavior', () => {
-    it('should render a download file button', () => {
-      const { selectors } = subject();
-      const { vaAlert, vaLink, vaLoadingIndicator } = selectors();
-      expect(vaLink).to.exist;
-      expect(vaAlert).to.not.exist;
-      expect(vaLoadingIndicator).to.not.exist;
-    });
+    apiRequestStub.restore();
+    recordEventStub.restore();
   });
 
   context('when the download button has been clicked', () => {
+    const createStubObject = () =>
+      sinon.stub(URL, 'createObjectURL').returns('my_stubbed_url.com');
+    const revokeObjectstub = () => sinon.stub(URL, 'revokeObjectURL');
+
     const triggerError = ({ link, status = '503' }) => {
       apiRequestStub.rejects({ errors: [{ status }] });
       fireEvent.click(link);
     };
 
-    it('should record the correct event when the request succeeds', async () => {
-      const { selectors } = subject();
-      const { vaLink: link } = selectors();
-      const event = 'caregivers-pdf-download--success';
-
-      apiRequestStub.resolves({
+    const triggerSuccess = ({ link }) => {
+      apiRequestStub.onFirstCall().resolves({
```

Comment:
I think unless we're looking for a specific call count, there is really no point in adding the `onFirstCall` here. You can just let it be `apiRequestStub.resolves()`.